jupyter_accessibility/dodo.py

```diff
@@ -0,0 +1,136 @@
+from pathlib import Path
+from shutil import copytree
+
+from pydantic import BaseModel, Field
+
+A11Y = Path().parent
+DOIT_CONFIG = dict(verbosity=2, list=dict(status=True, subtasks=True))
+
+
+def do(*args, cwd=A11Y, **kwargs):
+    """wrap a Action for consistency"""
+    from os import environ
+    from shlex import split
+
+    from doit.tools import CmdAction
+
+    if len(args) == 1:
+        args = split(args[0])
+    kwargs.setdefault("env", {})
+    kwargs["env"] = {**kwargs["env"], **environ}
```

Comment:
oh yea this logic is flipped. its kind of ok because it is remnant from a while ago when i was having issues in nested conda environments. will fix. good looks